modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Blogs/IBlogPostRepository.cs

```diff
@@ -17,5 +15,7 @@ public interface IBlogPostRepository : IBasicRepository<BlogPost, Guid>
         Task<List<BlogPost>> GetPagedListAsync(Guid blogId, int skipCount, int maxResultCount, string sorting, bool includeDetails = false, CancellationToken cancellationToken = default);
 
         Task<int> GetCountAsync(Guid blogId, CancellationToken cancellationToken = default);
+        
+        Task<bool> PostExistsAsync(Guid blogId, CancellationToken cancellationToken = default);
```

Comment:
I'm not sure about method name? This class is already `BlogPostRepository`, so can we use just `ExistsAsync` or `AnyAsync`?

Reply:
Right, deleted.